index.html

```diff
@@ -939,6 +939,20 @@ <h4>createDisclosureData</h4>
 |signature|,  the |bbsSignature|, |header| is the |bbsHeader|,
 |ph| is the |presentationHeader|,
 |messages| is |bbsMessages|, |disclosed_indexes| is |selectiveIndexes|,
+This operation takes as inputs
+   the                  original issuers public key as |PK|,
+   the                               |bbsSignature| as |signature|,
+   the                                  |bbsHeader| as |header|,
+   the                         |presentationHeader| as |ph|,
+   the                                |bbsMessages| as |messages|,
+   the                           |selectiveIndexes| as |disclosed_indexes|,
+   a                                                   |nym_secret|,
+   a                                                   |nym_domain|,
+   the only value of the |committed_messages| array as |holder_secret|,
+   and a                                               |secret_prover_blind|.
+In addition to providing
+the raw cryptographic proof value which is assigned to |bbsProof|,
+it also returns the |pseudonym|.
 |nym_secret|, |nym_domain|, the |committed_messages| array containing the
 |holder_secret| as its only value, and |secret_prover_blind|. In addition to
```

Comment:
Inadvertently duplicated text.
```suggestion
In addition to
```